src/Config.php

```diff
@@ -187,21 +187,15 @@ private function setSender($c)
             $default = "Rollbar\Senders\FluentSender";
 
             if (isset($c['fluent_host'])) {
-                $c['senderOptions'] = array(
-                    'fluentHost' => $c['fluent_host']
-                );
+                $c['senderOptions']['fluentHost'] = $c['fluent_host'];
             }
 
             if (isset($c['fluent_port'])) {
-                $c['senderOptions'] = array(
-                    'fluentPort' => $c['fluent_port']
-                );
+                $c['senderOptions']['fluentPort'] = $c['fluent_port'];
             }
 
             if (isset($c['fluent_tag'])) {
-                $c['senderOptions'] = array(
-                    'fluentTag' => $c['fluent_tag']
-                );
+                $c['senderOptions']['fluentTag'] = $c['fluent_tag'];
```

Comment:
Replying to all above: No, because it would override the sender config. The current impl extends each of the fields by another setting.